packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -98,6 +99,33 @@ export const styles = css`
         outline: none;
     }
 
+    .control::before {
+        content: '';
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        pointer-events: none;
+        box-sizing: border-box;
+        outline: 0px solid transparent;
+        color: transparent;
+        background-clip: content-box;
+        z-index: -1;
+    }
+
+    .control${focusVisible}::before {
```

Comment:
In the latest storybook build I'm not seeing the double border for focus visible any more. Was that an intentional change?

Reply:
Nope, this issue is also caused by the z-index being -1. I'm looking into fixes now.

Reply:
My newest commit fixes this issue but requires modifying the storybook.ts file which is used for all matrix stories. I double checked the matrix stories that use z-index as well as the chromatic tests for all matrix stories, and everything seems to be fine. This is the simplest way I could find to fix this issue, but we might need to discuss it with the team to make sure that everyone's okay with it.

Reply:
Yeah, I'm nervous about that approach. Anything that requires global storybook changes to render correctly might also require client apps to make similar changes, and we don't want them to have to do anything special to use our components.

I don't think I'll have a chance to brainstorm better options this week (I'm only working a few hours) but you could try with Mert or put this aside and work on your other projects until people are back next week.

Reply:
My newest commit limits the change I previously made to the storybook.ts file to just the button, toggle button, and anchor button matrix story file. 👍

Reply:
I'm seeing that for the buttons in the Docs page, (1) they don't get a double outline when focused, and (2) they don't get a green fill when pressed/active. I'm guessing that's also related to the z-index issue.